public/api-merged.json

```diff
@@ -22,25 +22,25 @@
   },
   "basePath": "/api",
   "paths": {
-    "/admin/ldap/reload": {
-      "post": {
-        "security": [
+    "/access-control/roles": {
+      "get": {
+        "description": "Gets all existing roles. The response contains all global and organization local roles, for the organization which user is signed in.\n\nYou need to have a permission with action `roles:read` and scope `roles:*`.",
```

Comment:
I am not sure I understand the sentence from global and organization ....

Reply:
I don't know much about the RBAC roles either: it's generated from [this description](https://github.com/grafana/grafana-enterprise/blob/36a002e3a25c5117a6e893505305527805818e50/src/pkg/extensions/accesscontrol/acimpl/api.go#L110) which was based on the [docs](https://grafana.com/docs/grafana/latest/developers/http_api/access_control/#get-all-roles)